### ISSUE TITLE
Font issues

### ISSUE THREAD

**Issue by samokosik** [ONLYOFFICE/DesktopEditors#769]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Linux (DEB package)

### Version information

 6.4.2.6

### Expected Behavior

Having a font which can display characters, such as slovak ones.

### Actual Behavior

I installed onlyoffice and I have a problem with non ASCII characters - when a non-ASCII character is in the interface, it is substituted with another one which ends up in oddly-looking texts: 
https://imgur.com/ForXTP8.png

This does not happen within other apps.

**Comment by ShockwaveNN** [comment]
Sorry, I don't fully understand you and probably because I have no understanding of language on screenshot

Could you describe how to reproduce your issue, step by step

Which Linux distribution you've using, which system language and etc

**Comment by samokosik** [comment]
Right, sorry for a lack of information. Here are listed the things which were needed.
The language is Slovak, characters such as "ˇščťž" were from that language and those are the ones incorrectly displayed (or better said, a different font is used to display them).

My system is Arch Linux with i3 gaps window manager, language is set to English but since Slovak is my first language, I also need to use that one. Locales are set to Slovak, as well and other applications have no issues.

Regarding the reproduction of the issue, there is not much to add - I have just installed onlyoffce-bin package from the AUR (https://aur.archlinux.org/packages/onlyoffice-bin/) and the problem has been there ever since

**Comment by ShockwaveNN** [comment]
Could you recheck this problem on other systems? Because we use mostly Ubuntu and officially we do not provide this package you've mention, so I think it will simplify for use - understand is this problem only for Arch Aur package or even on our default installation

And also you can try to check flatpak/snap/AppImage

**Comment by samokosik** [comment]
Hey, I tried it on kali and noticed a different font usage. The font which was on kali supports mentioned characters.

**Comment by ShockwaveNN** [comment]
In that case I think it's problem with installation on arch and since it's not our official package, but made by community - better contact authors of this package

**Comment by samokosik** [comment]
Are you aware of who those maintainers are?

**Comment by ShockwaveNN** [comment]
Sorry, no, I think you should find some details at https://aur.archlinux.org/packages/onlyoffice-bin/
